index.html

```diff
@@ -58,7 +58,207 @@ <h4>LET'S CONNECT</h4>
+                <h1>About Myself</h1>
+            </article>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <h4>LET'S CONNECT</h4>
+            <ul>
+                <li>
+                    <a title="The link will take you to my facebook profile" href="https://www.facebook.com/Mpdaddy/">
+                    <i class="fa fa-facebook social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my twitter profile" href="https://github.com/MPDADDY">
+                        <i class="fa fa-twitter social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my instagram profile" href="https://www.instagram.com/mpdaddympd/">
+                        <i class="fa fa-instagram social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my linkin profile" href="https://www.linkedin.com/in/anold-mwawasi">
+                        <i class="fa fa-linkedin social"></i>
+                    </a>
+                </li>
+            </ul>
+            <div class="getresume">
+                <a title="get my resume" href="#">Get my resume</a>
+            </div>
+            <article id="what_i_do">
+              <ul class="gridcolumn">
+                <li>
+                    <div class="headlang marginleft">
+                        <h5>Languages</h5>
+                        <i class="fa fa-angle-down fontarrows"></i>
+                    </div>
+                    <div class="langcontainer">
+                        <div class="langgrid">
+                            <img src="images/Ellipse 1.png" alt="">
+                            <p>javascript</p>
+                        </div>
+                        <div class="langgrid">
+                            <img src="images/Ellipse 1 (1).png" alt="">
+                            <p>HTML</p>
+                        </div>
+                        <div class="langgrid">
+                            <img src="images/Ellipse 1 (2).png" alt="">
+                            <p>CSS</p>
+                        </div>
+                    </div>
+                </li>
+                <li>
+                    <div class="headlang">
+                        <h5>Frameworks</h5>
+                        <i class="fa fa-angle-down fontarrows"></i>
+                    </div>
+                </li>
+                <li>
+                    <div class="headlang">
+                        <h5>Skills</h5>
+                        <i class="fa fa-angle-down fontarrows"></i>
+                    </div>
+                </li>
+              </ul>
```

Comment:
Good work implementing an `ul` tag with a list of items, since the `ul` tag is for listing items, it's not necessary to use a div inside it. The `li` works as a `div` so you will save a line of code and it will be DRY
To do this, you could replace your div tag class with your li tag, it shouldn't be hard to make this change and your code will look better! 💯